app/schemas/models/user.coffee

```diff
@@ -154,6 +154,7 @@ _.extend UserSchema.properties,
       descipription: 'Player customization options, including hero name, objectId and applied color tints.'
     }, {
       thangType: c.objectId(links: [{rel: 'db', href: '/db/thang.type/{($)}/version'}], title: 'Thang Type', description: 'The ThangType of the hero.', format: 'thang-type'),
+      playerHeroName: c.shortString({ title: 'Ozaria Hero Name', description: 'The user set name for the ozaria hero. Used in cinematics.' })
```

Comment:
Out of curiosity, is this something that users will set from character customization screen?

Reply:
Yes!